roles/openshift_cfme/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+openshift_cfme_project: cfme2
+openshift_cfme_project_description: ManageIQ - CloudForms Management Engine
+openshift_cfme_user: cfme2
+openshift_cfme_service_account: "system:serviceaccount:{{ openshift_cfme_project }}:default"
+openshift_cfme_pv_templates:
+  - miq-pv-db
+  - miq-pv-region
+  - miq-pv-server
+openshift_cfme_maxImagesBulkImportedPerRepository: 100
+openshift_cfme_nfs_server: "{{ groups.nfs.0 }}"
```

Comment:
Until I `include` the `std_include.yml` file I have to stick with basic ansible names for references. Later on these names, like `groups.nfs.0` and `groups.masters.0` will be replaced with their appropriate `oo_*` equivalents, e.g. `groups.masters.0` → `oo_first_master`.